### PR TITLE
This replaces toolkit/page-headings with govuk frontend page titles

### PR DESCRIPTION
**Pull request by agalamatis**
trello: https://trello.com/c/9LvUzYaP/158-2-update-page-headings-to-govuk-frontend-style-in-supplier-frontend

### CODE REVIEW

File: app/templates/frameworks/contract_review.html [hunk 1]
```diff
@@ -37,6 +37,7 @@
     %}
       {% include "toolkit/page-heading.html" %}
     {% endwith %}
```
Comment (lfdebrux) [inline app/templates/frameworks/contract_review.html]: Lines above need to be removed

**Comment by lfdebrux** [comment]
Appears to be a problem with spacing and banners on the framework dashboard page, even though the override has been added correctly 🤔 

image

Any thoughts @aliuk2012?

**Comment by agalamatis** [comment]
@lfdebrux / @aliuk2012 I think I found the issue, fix in the last commit.

**Comment by lfdebrux** [comment]
> @lfdebrux / @aliuk2012 I think I found the issue, fix in the last commit.

Ah, good spot, unfortunately the issue is still there (because the banner is inside the #content div).

**Comment by lfdebrux** [comment]
Missed one here also 

https://github.com/alphagov/digitalmarketplace-supplier-frontend/pull/1139/files#diff-d52d46ec4d8bd41c04744e59862dce32R93-R99

**Comment by agalamatis** [comment]
> Missed one here also
> 
> https://github.com/alphagov/digitalmarketplace-supplier-frontend/pull/1139/files#diff-d52d46ec4d8bd41c04744e59862dce32R93-R99

fixed, pushed

**Comment by lfdebrux** [comment]
> > @lfdebrux / @aliuk2012 I think I found the issue, fix in the last commit.
> 
> Ah, good spot, unfortunately the issue is still there (because the banner is inside the #content div).

Looks like we can apply patch https://github.com/alphagov/digitalmarketplace-admin-frontend/commit/52e9b1550369ddfaaaa8603d8b60940c7a176f66 from admin frontend to fix the notification banners

**Pull request merged**